src/net.rs

```diff
@@ -965,7 +965,8 @@ impl AcceptAddrsBuf {
     }
 
     fn args(&self) -> (PVOID, DWORD, DWORD, DWORD) {
-        let remote_offset = unsafe { &(*(0 as *const AcceptAddrsBuf)).remote as *const _ as usize };
+        let buf: *const AcceptAddrsBuf = unsafe { mem::zeroed() };
+        let remote_offset = unsafe { &(*buf).remote as *const _ as usize };
```

Comment:
@kennykerr this stops the warning, but I'm not sure if it's _correct_. Can you take a look?

Reply:
The code appears to be equivalent.